cernopendata/modules/fixtures/data/records/lhcb-antimatter-matters-2017.json

```diff
@@ -0,0 +1,196 @@
+      {
+        "description": "The LHCb experiment is described in",
+        "url": "http://cds.cern.ch/record/1129809"
+      }
+    ]
+  },
+  "accelerator": "CERN-LHC",
+  "collaboration": {
+    "name": "LHCb collaboration"
+  },
+  "collections": [
+    "LHCb-Derived-Datasets"
+  ],
+  "date_published": "2017",
+  "distribution": {
+    "formats": [
+      "root"
+    ], 
+    "number_events": 8550000,
+    "number_files": 3,
+    "size": 1113480280
+  },
+  "doi": "10.7483/OPENDATA.LHCB.AOF7.JH09",
+  "experiment": "LHCb",
+    "files": [
+    {
+      "size": 666484974, 
+      "type": "xrootd", 
+      "uri": "root://eospublic.cern.ch//eos/opendata/lhcb/AntimatterMatters2017/data/B2HHH_MagnetDown.root"
+    }, 
+    {
+      "size": 444723234, 
+      "type": "xrootd", 
+      "uri": "root://eospublic.cern.ch//eos/opendata/lhcb/AntimatterMatters2017/data/B2HHH_MagnetUp.root"
+    }, 
+    {
+      "size": 2272072, 
+      "type": "xrootd", 
+      "uri": "root://eospublic.cern.ch//eos/opendata/lhcb/AntimatterMatters2017/data/PhaseSpaceSimulation.root"
+    }
+  ], 
+  "methodology": {
+    "description": "Offline Selections cuts were made on: track transverse momentum (pT), sum of pT of Tracks, Track momentum, B candidate mass assuming all tracks are Kaons, Track impact parameter chi-square, sum of impact parameter chi-square of tracks, B candidate vertex fit chi-square."
+  },
+  "publisher": "CERN Open Data Portal",
+  "recid": "FIXME",
+  "title": "Matter Antimatter Differences (B meson decays to three hadrons) - Data Files",
+  "type": {
+    "primary": "Dataset",
+    "secondary": []
+  },
+  "usage": {
+    "description": "The data files can be accessed using ROOT.",
+    "links": [
+      {
+        "url": "https://root.cern.ch/"
+      }
+    ]
+  },
+  "use_with": {
+    "description": "The data files are expected to be used with the Project Notebook.",
+    "links": [
+      {
+        "recid": "FIXME_Software-record"
+      }
+    ]
+  },
+  "validation": {
+    "description": "The LHCb collaboration performs online and offline monitoring of the quality of the data collected. Only data for which the primary detector subsystems are highly performant are used for analysis.",
+    "links": [
+      {
+        "description": "The performance of the detector is described in", 
+        "url": "https://cds.cern.ch/record/1978280?ln=en"
+      }
+    ]
+  }
+}
+,
+{
+  "abstract": {
+    "description": "This documentation provides a basic description of the contents of the project and instructions on how to get started."
+  },
+  "accelerator": "CERN-LHC",
+  "collaboration": {
+    "name": "LHCb collaboration"
+  },
+  "collections": [
+    "LHCb-Learning-Resources"
+  ],
+  "date_published": "2017",
+  "distribution": {
+    "formats": [
+      "pdf"
+    ], 
+    "number_files": 1,
+    "size": 496943
+  },
+  "doi": "10.7483/OPENDATA.LHCB.IID3.87CV",
+  "experiment": "LHCb",
+  "files": [
+    {
+      "size": 496943, 
```

Comment:
"type": "xrootd" as we do for datasets?!

Reply:
For the documentation file? It's just a .pdf.

Reply:
Yes, but it's stored on EOS, accessible via XRootD, see the `uri` property.  

Perhaps we can simply not store `xrootd` in "type",  since this will be true for most files... actually all except the file indexes. Not sure whether there is a value in storing this...

Reply:
Okay, I will use xrootd for now